src/com/owncloud/android/operations/RenameFileOperation.java

```diff
@@ -168,7 +169,7 @@ private void saveLocalFile() {
                 // notify MediaScanner about removed file
                 getStorageManager().deleteFileInMediaScan(oldPath);
                 // notify to scan about new file
-                getStorageManager().triggerMediaScan(newPath);
+                FileDataStorageManager.triggerMediaScan(newPath);
```

Comment:
Please delete the static label from triggerMediaScan and keep calling it with getStorageManager().triggerMediaScan(newPath);